server-spi-private/src/main/java/org/keycloak/models/Constants.java

```diff
@@ -37,6 +37,9 @@ public final class Constants {
     public static final String BROKER_SERVICE_CLIENT_ID = "broker";
     public static final String REALM_MANAGEMENT_CLIENT_ID = "realm-management";
 
+    public static final String ACCOUNT_RESOURCE_PROVIDER_KEY = "accountResourceProvider";
```

Comment:
This doesn't belong here, and isn't clear it's a theme properties key. Maybe just add to Theme.java, or create a ThemeProperties.java?

Reply:
Created a ThemeProperties file.